client/dataset_api_client.go

```diff
@@ -141,8 +141,8 @@ func (cli *DatasetAPIClient) AddEventToInstance(instanceID string, e *model.Even
 	}
 	defer resp.Body.Close()
 
-	if resp.StatusCode != http.StatusCreated {
```

Comment:
Do you know if this change on the dataset API was intentional? I just checked the swagger spec and it still specifies 201. I am wondering if it was made by accident, or if the swagger spec was just not updated.

Reply:
I think the dataset API is wrong (the spec is right), and will be fixed (Nathan says "soon").

I should backout this change, so that it will work with a fixed API.

Clearly, we're not using this reporter much, and I could be considered over-keen in wanting testing to work...

Reply:
done